app/index.js

```diff
@@ -90,6 +90,15 @@ new mediator.TerminationConditionModelerHook(terminationConditionModeler);
 const errorBar = new ErrorBar(document.getElementById("errorBar"), mediator);
 const checker = new Checker(mediator, errorBar);
 
+// construction Mode for User Study, set to false for normal mode
+const constructionMode = false;
+
+const titleObjectiveModel = document.getElementById("titleObjectiveModel");
+titleObjectiveModel.innerHTML = constructionMode ? "Milestone" : "Objective Model";
```

Comment:
Please please please use normal if-statement because they are much more readable. I beg you